elastic.js/elastic.js.d.ts

```diff
@@ -40,7 +40,7 @@ declare module elasticjs {
      Retrieves the internal agg object. This is typically used by
      internal API functions so use with caution.
      */
-    toJSON(): String;
+    toJSON(): Object;
```

Comment:
@vvakame , thank you for review! Type "Object" has been changed to "any". Please review.